node/rpc_test.go

```diff
@@ -25,11 +24,34 @@ import (
 // celestia-node. They will be removed upon refactoring of the RPC
 // architecture and Public API. @renaynay @Wondertan.
 
+const testHeight = uint64(2)
```

Comment:
I would rather make this part of the test-case struct:
```go
	[]struct {
		nID         string
		height      uint64
		expectedErr bool
		errMsg      string
	}
```
 But can happen in a follow-up PR. We should not only test with height 2.

Reply:
We can't query for actual data, as we're not able to `/submit_pfd` - results are empty, so it doesn't matter what block we check.
This is more like a "smoke test" it only checks that nothing terrible happens when RPC is called.